code/game/objects/items/melee/energy.dm

```diff
@@ -63,224 +83,270 @@
 	playsound(loc, hitsound, get_clamped_volume(), TRUE, -1)
 	add_fingerprint(user)
 
-/obj/item/melee/transforming/energy/axe
+/*
+ * Signal proc for [COMSIG_TRANSFORMING_ON_TRANSFORM].
+ *
+ * Updates our icon to have the correct color,
+ * updates the amount of heat our item gives out,
+ * enables / disables embedding, and
+ * starts / stops processing.
+ *
+ * Also gives feedback to the user and activates or deactives the glow.
+ */
+/obj/item/melee/energy/proc/on_transform(obj/item/source, mob/user, active)
+	SIGNAL_HANDLER
+
+	blade_active = active
+	if(active)
+		if(sword_color_icon)
+			icon_state = "[icon_state]_[sword_color_icon]"
+		if(embedding)
+			updateEmbedding()
+		heat = active_heat
+		START_PROCESSING(SSobj, src)
+	else
+		if(embedding)
+			disableEmbedding()
+		heat = initial(heat)
+		STOP_PROCESSING(SSobj, src)
+
+	balloon_alert(user, "[name] [active ? "enabled":"disabled"]")
+	playsound(user ? user : src, active ? 'sound/weapons/saberon.ogg' : 'sound/weapons/saberoff.ogg', 35, TRUE)
+	set_light_on(active)
+	return COMPONENT_NO_DEFAULT_MESSAGE
+
+/// Energy axe - extremely strong.
+/obj/item/melee/energy/axe
 	name = "energy axe"
 	desc = "An energized battle axe."
-	icon_state = "axe0"
+	icon_state = "axe"
 	lefthand_file = 'icons/mob/inhands/weapons/axes_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/axes_righthand.dmi'
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("attacks", "chops", "cleaves", "tears", "lacerates", "cuts")
+	attack_verb_simple = list("attack", "chop", "cleave", "tear", "lacerate", "cut")
 	force = 40
-	force_on = 150
 	throwforce = 25
-	throwforce_on = 30
-	hitsound = 'sound/weapons/bladeslice.ogg'
 	throw_speed = 3
 	throw_range = 5
+	armour_penetration = 100
+	sharpness = IS_SHARP
 	w_class = WEIGHT_CLASS_NORMAL
-	w_class_on = WEIGHT_CLASS_HUGE
 	flags_1 = CONDUCT_1
-	armour_penetration = 100
-	attack_verb_off = list("attacks", "chops", "cleaves", "tears", "lacerates", "cuts")
-	attack_verb_on = list()
-	light_color = "#40ceff"
-
-/obj/item/melee/transforming/energy/axe/suicide_act(mob/living/user)
+	light_color = LIGHT_COLOR_LIGHT_CYAN
+
+	active_force = 150
+	active_throwforce = 30
+	active_w_class = WEIGHT_CLASS_HUGE
+
+/obj/item/melee/energy/axe/make_transformable()
+	AddComponent(/datum/component/transforming, \
+		force_on = active_force, \
+		throwforce_on = active_throwforce, \
+		throw_speed_on = throw_speed, \
+		sharpness_on = sharpness, \
+		w_class_on = active_w_class)
```

Comment:
No bleedforce or attack verbs/sounds?

Reply:
its a debug item that instantly disemembers carbons. Its not really on the same playing field as other energy weapons